presto-main/src/main/java/com/facebook/presto/cost/FilterStatsCalculator.java

```diff
@@ -108,6 +109,17 @@ else if (node.getLeft() instanceof Literal && node.getRight() instanceof SymbolR
             }
         }
 
+        @Override
```

Comment:
Also fixup. And we want to unit test it.
Testing can probably  be an effort done together with other expression types already supported by `FilterStatsCalculator`.